Azure/sonic-buildimage
# [sai.profile] move hwsku specific device configs to device/mellanox/p…

…latform/hwsku



**- What I did**
Changed the approach of managing configuration files, placing them under device/<vendor>/<platform>/<HwSku>
This way we no longer have to check them into sai package.
**- How I did it**
- Checked in configuration files under device hwsku folder
- Changed sai.profile files to point to matching configuration files
**- How to verify it**
Deploy on a switch, make sure that syncd docker has a matching config file under /usr/share/sonic/hwsku, and switch is configured properly.
**- Description for the changelog**


**- A picture of a cute animal (not mandatory but encouraged)**
